lib/web_ui/lib/src/engine/pointer_binding/event_position_helper.dart

```diff
@@ -25,6 +29,12 @@ ui.Offset computeEventOffsetToTarget(DomMouseEvent event, DomElement actualTarge
   }
 
   final bool isTargetOutsideOfShadowDOM = event.target != actualTarget;
+  final bool isInput = event.target.toString() == 'input';
+
+  if(isInput) {
+    return _computeOffsetForInputs(event);
+  }
```

Comment:
We should also check that `geometry` is available.
```suggestion
  if (isInput) {
    final EditableTextGeomtery inputGeometry = textEditing.strategy.geometry;
    if (inputGeometry != null) {
      return _computeOffsetForInputs(event, inputGeometry);
    }
  }
```